placeholder_accessors/index.md

```diff
@@ -199,8 +199,29 @@ Another method, that allows specifying the memory object the placeholder
 accessor will be associated is also provided:
 
 `handler::require(buffer<T, dim> b,
-    accessor<T, dim, mode, target, acess::placeholder::true_t>)`
+    accessor<T, dim, mode, target, access::placeholder::true_t>)`
 
+### `accessor`s without buffers
+
+If a placeholder accessor is not tied to a buffer within a command group, then
```

Comment:
`If a placeholder accessor` -> `If a placeholder accessor which was not constructed with a buffer`.